config/webpack.common.config.js

```diff
@@ -132,21 +132,11 @@ const commonWebpackConfig = {
 
   resolve: {
     extensions: ['.ts', '.tsx', '.js'],
+    modules: ['node_modules'],
     alias: {
-      "@terrestris/react-geo": path.join(__dirname + '/../', 'node_modules', '@terrestris/react-geo'),
-      "@terrestris/base-util": path.join(__dirname + '/../', 'node_modules', '@terrestris/base-util'),
-      "@terrestris/ol-util": path.join(__dirname + '/../', 'node_modules', '@terrestris/ol-util'),
-      antd: path.join(__dirname + '/../', 'node_modules', 'antd'),
-      baseclient: path.join(__dirname + '/../', 'src'),
-      "baseclient-components": path.join(__dirname + '/../../', 'baseclient-components'),
-      "baseclient-state": path.join(__dirname + '/../../', 'baseclient-state'),
-      ol: path.join(__dirname + '/../', 'node_modules', 'ol'),
-      react: path.join(__dirname + '/../', 'node_modules', 'react'),
-      "react-redux": path.join(__dirname + '/../', 'node_modules', 'react-redux'),
-      "react-i18next": path.join(__dirname + '/../', 'node_modules', 'react-i18next'),
-      tslib: path.join(__dirname + '/../', 'node_modules', 'tslib'),
-    },
-    modules: ['node_modules', '../baseclient/node_modules', './packages/baseclient/node_modules']
+      // This is need if working with npm link
+      'ol': path.join(__dirname, '../node_modules', 'ol')
```

Comment:
Please comment this line as it should only needed when working with a linked `@terrestris/react-geo` package.

This should then added as a note the the README (under section `Development`).